Source/Make.WarpX

```diff
@@ -61,6 +61,7 @@ endif
 include $(WARPX_HOME)/Source/Make.package
 include $(WARPX_HOME)/Source/BoundaryConditions/Make.package
 include $(WARPX_HOME)/Source/Diagnostics/Make.package
+include $(WARPX_HOME)/Source/Diagnostics/ReducedDiags/Make.package
```

Comment:
Could you move this part to `Source/Diagnostics/Make.package `, as done for `Source/Particles/Make.package`?